[2025-10-14T18:45:13.766+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pyspark_project_pipline.create_topic_sensors_sample manual__2025-10-14T18:44:36.290868+00:00 [queued]>
[2025-10-14T18:45:13.773+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pyspark_project_pipline.create_topic_sensors_sample manual__2025-10-14T18:44:36.290868+00:00 [queued]>
[2025-10-14T18:45:13.773+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2025-10-14T18:45:13.790+0000] {taskinstance.py:2217} INFO - Executing <Task(_PythonDecoratedOperator): create_topic_sensors_sample> on 2025-10-14 18:44:36.290868+00:00
[2025-10-14T18:45:13.795+0000] {standard_task_runner.py:60} INFO - Started process 938 to run task
[2025-10-14T18:45:13.798+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'pyspark_project_pipline', 'create_topic_sensors_sample', 'manual__2025-10-14T18:44:36.290868+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/pyspark_project_dims.py', '--cfg-path', '/tmp/tmp4z9_w6_s']
[2025-10-14T18:45:13.800+0000] {standard_task_runner.py:88} INFO - Job 21: Subtask create_topic_sensors_sample
[2025-10-14T18:45:13.845+0000] {task_command.py:423} INFO - Running <TaskInstance: pyspark_project_pipline.create_topic_sensors_sample manual__2025-10-14T18:44:36.290868+00:00 [running]> on host f807c778cd29
[2025-10-14T18:45:13.911+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pyspark_project_pipline' AIRFLOW_CTX_TASK_ID='create_topic_sensors_sample' AIRFLOW_CTX_EXECUTION_DATE='2025-10-14T18:44:36.290868+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-14T18:44:36.290868+00:00'
[2025-10-14T18:45:15.620+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 444, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 414, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/base.py", line 241, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 200, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 217, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/pyspark_project_dims.py", line 134, in create_topic_sensors_sample
    create_kafka_topic("sensors-sample")
  File "/opt/airflow/dags/pyspark_project_dims.py", line 78, in create_kafka_topic
    existing_topics = [t.strip() for t in topics_list.splitlins()]
AttributeError: 'CompletedProcess' object has no attribute 'splitlins'
[2025-10-14T18:45:15.632+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=pyspark_project_pipline, task_id=create_topic_sensors_sample, execution_date=20251014T184436, start_date=20251014T184513, end_date=20251014T184515
[2025-10-14T18:45:15.643+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 21 for task create_topic_sensors_sample ('CompletedProcess' object has no attribute 'splitlins'; 938)
[2025-10-14T18:45:15.657+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2025-10-14T18:45:15.675+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
